bundles/org.openhab.ui/web/src/components/thing/channel-link.vue

```diff
@@ -44,6 +44,7 @@
       <f7-icon slot="media" color="green" aurora="f7:plus_circle_fill" ios="f7:plus_circle_fill" md="material:control_point" />
     </f7-list-item>
     <f7-list-button class="searchbar-ignore" color="blue" :title="(channelType.parameterGroups.length || channelType.parameters.length) ? 'Configure Channel' : 'Channel Details'" @click="configureChannel()" />
+    <f7-list-button class="searchbar-ignore" color="blue" :title="(channelType.parameterGroups.length || channelType.parameters.length) ? 'Copy Channel' : ''" @click="copyChannel()" />
```

Comment:
Since not all Things are extensible (not all Things support the addition of channels, e.g. Astro Things), the copy option should only be shown when the Thing is extensible. BTW you can see that the remove button is also only shown in that case.

```suggestion
    <f7-list-button class="searchbar-ignore" v-if="extensible" color="blue" :title="(channelType.parameterGroups.length || channelType.parameters.length) ? 'Copy Channel' : ''" @click="copyChannel()" />
```